site2/docs/security-tls-authentication.md

```diff
@@ -75,7 +75,7 @@ superUserRoles=admin
 # Authentication settings of the broker itself. Used when the broker connects to other brokers, either in same or other clusters
 brokerClientTlsEnabled=true
 brokerClientAuthenticationPlugin=org.apache.pulsar.client.impl.auth.AuthenticationTls
-brokerClientAuthenticationParameters={"tlsCertFile":"/path/my-ca/admin.cert.pem,tlsKeyFile:/path/my-ca/admin.key-pk8.pem"}
+brokerClientAuthenticationParameters={"tlsCertFile":"/path/my-ca/broker.cert.pem,tlsKeyFile:/path/my-ca/broker.key-pk8.pem"}
```

Comment:
`brokerClientAuthenticationParameters` is used for the client in the broker, it just needs a superuser role, so it's ok to use admin cert. In this example, we make the admin cert as a superuser, so it should be correct for this setting. 

Any issue when you trying to set up with the document?